shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-sqlserver/src/main/java/org/apache/shardingsphere/sql/parser/sqlserver/visitor/SQLServerVisitor.java

```diff
@@ -430,11 +431,11 @@ public final ASTNode visitAggregationFunction(final AggregationFunctionContext c
     
     private ASTNode createAggregationSegment(final AggregationFunctionContext ctx, final String aggregationType) {
         AggregationType type = AggregationType.valueOf(aggregationType.toUpperCase());
-        int innerExpressionStartIndex = ((TerminalNode) ctx.getChild(1)).getSymbol().getStartIndex();
+        String innerExpression = ctx.start.getInputStream().getText(new Interval(((TerminalNode) ctx.getChild(1)).getSymbol().getStartIndex(), ctx.stop.getStopIndex()));
```

Comment:
The result of `ctx.start.getInputStream().getText()` and `ctx.getText()` are the same?

Reply:
Do you think `ctx.getChild(1)).getSymbol().getStartIndex()` == `ctx.LP_().getSymbol().getStartIndex()`?

From the definition of `aggregationFunction `,
```xml
aggregationFunction
    : aggregationFunctionName LP_ distinct? (expr (COMMA_ expr)* | ASTERISK_)? RP_
```
What we want is from LP_ to RP_, right?

Reply:
> The result of `ctx.start.getInputStream().getText()` and `ctx.getText()` are the same?

@tristaZero According to the documentation of the `ctx.getText()` method, we can find that the method does not return whitespace.
```
/** 
  *  Return the combined text of all leaf nodes. Does not get any
  *  off-channel tokens (if any) so won't return whitespace and
  *  comments if they are sent to parser on hidden channel.
  */
String getText();
```
In order to get the original sql that contains whitespace, we need to use the `ctx.start.getInputStream().getText()` method to specify the start and end interval through Interval.

Reply:
> Do you think ctx.getChild(1)).getSymbol().getStartIndex() == ctx.LP_().getSymbol().getStartIndex()?

Yes, maybe using `ctx.LP_().getSymbol().getStartIndex()` is more easier to read. I will optimise it.

> From the definition of aggregationFunction ,
> 
> aggregationFunction
>     : aggregationFunctionName LP_ distinct? (expr (COMMA_ expr)* | ASTERISK_)? RP_
> What we want is from LP_ to RP_, right?

Yes, the innerExpression include sql string from LP_ to RP_. The previous innerExpressionStartIndex is also the same logic.